File: src/main/java/com/trade/securities/application/port/in/GetCommodityTradingInfoUseCase.java
package com.trade.securities.application.port.in;

import com.trade.securities.domain.CommodityTradingInfo;
import java.util.List;

public interface GetCommodityTradingInfoUseCase {
    List<CommodityTradingInfo> getCommodityTradingInfo(String marketType, String basDd);
}

File: src/main/kotlin/com/trade/securities/application/port/in/GetCommodityTradingInfoUseCase.kt
package com.trade.securities.application.port.`in`

import com.trade.securities.domain.CommodityTradingInfo

interface GetCommodityTradingInfoUseCase {
    fun getCommodityTradingInfo(marketType: String, basDd: String): List<CommodityTradingInfo>
}
